projects/angular.dev/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable: ~
+versions:
+  github: angular/angular
+dependencies:
+  nodejs.org: ^20
+build:
+  dependencies:
+    npmjs.com: ^10
+    curl.se: '*'
+  script:
+    - curl -L https://registry.npmjs.org/@angular/cli/-/cli-{{version}}.tgz --output cli-{{version}}.tgz
```

Comment:
why can't this url be used as distributable? you should be able to use `.` as the path once its upacked, i believe.

Reply:
It looks like #3561 
There I was getting an error when using `.`

Reply:
But I'll try now